src/lesson11/task2/CustomException.java

```diff
@@ -0,0 +1,13 @@
+package lesson11.task2;
+
+public class CustomException extends Exception{
+    private String message;
+    public CustomException(String message) {
+        super(message);
+        this.message = message;
```

Comment:
так як поле меседж є в батьківському класі то воно наслідується і є тут, тому його не потрібно вказувати та явно присвоювати достатно викликати батьківський конструктор super(message)
```suggestion
    public CustomException(String message) {
        super(message);
```